backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/ScheduledPublicationService.java

```diff
@@ -36,7 +36,8 @@ public ScheduledPublicationService(
     this.mailSender = mailSender;
   }
 
-  @Scheduled(fixedRateString = "PT1M") // Runs every minute
+  // Runs every minute, starts initially after 5s
+  @Scheduled(fixedRateString = "PT1M", initialDelayString = "PT5S")
```

Comment:
We now start the job right after the server starts (with a 5s delay). To me this makes sense in general and also reduces the integration test duration from 60s to 5s.